pkg/kube/client.go

```diff
@@ -273,14 +273,44 @@ func perform(infos ResourceList, fn func(*resource.Info) error) error {
 		return ErrNoObjectsVisited
 	}
 
-	for _, info := range infos {
-		if err := fn(info); err != nil {
+	errs := make(chan error)
+	go batchPerform(infos, fn, errs)
+
+	for range infos {
+		err := <-errs
+		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func batchPerform(infos Result, fn ResourceActorFunc, errs chan<- error) {
```

Comment:
@cgetzen As an FYI, `Result` has been renamed to `ResourceList`, which is what is causing your error

Reply:
better name! thanks